### DISCUSSION TITLE
Caught exception: Could not encode content to H

### DISCUSSION THREAD

**Discussion by GustelDE** [Bacon/BaconQrCode#168] [Q&A]
Hi, if  i try to use the error corrention, I always get:

**Caught exception: Could not encode content to H**

What can I do? Installed on a Rapi4, PHP8.2
And it doesn't matter if I select H, Q, M, L

**Comment by DASPRiD** [comment]
Sounds like you are wrongly passing the error correction parameter to the encoding parameter.
